backend/app/adapter/facebook/account.go

```diff
@@ -1,16 +1,21 @@
 package facebook
 
 import (
+	"net/http"
+	"net/url"
+
+	"github.com/short-d/app/fw"
 	"github.com/short-d/short/app/entity"
 	"github.com/short-d/short/app/usecase/service"
-
-	fb "github.com/huandu/facebook"
```

Comment:
Please run `go mod tidy` because we don't need this dependency anymore

Reply:
addressed comment